okio-files/src/commonTest/kotlin/okio/files/FileSystemTest.kt

```diff
@@ -46,45 +50,42 @@ class FileSystemTest {
   @Test
   fun `list no such directory`() {
     assertFailsWith<IOException> {
-      Filesystem.SYSTEM.list("/tmp/unlikely-directory/ce70dc67c24823e695e616145ce38403".toPath())
+      Filesystem.SYSTEM.list("$tmpDirectory/unlikely-directory/ce70dc67c24823e695e616145ce38403".toPath())
     }
   }
 
   @Test
   fun `file source no such directory`() {
     assertFailsWith<IOException> {
-      Filesystem.SYSTEM.source("/tmp/unlikely-directory/ce70dc67c24823e695e616145ce38403".toPath())
+      Filesystem.SYSTEM.source("$tmpDirectory/unlikely-directory/ce70dc67c24823e695e616145ce38403".toPath())
     }
   }
 
   @Test
   fun `file source`() {
     val source = Filesystem.SYSTEM.source("gradle.properties".toPath())
     val buffer = Buffer()
-    assertEquals(47L, source.read(buffer, 100L))
+    assertTrue(source.read(buffer, 100L) <= 49L) // either 47 on posix or 49 with \r\n line feeds on windows
```

Comment:
For this line feed stuff I'd prefer we do a replaceAll to get rid of them before asserting.

Reply:
Yup, much better and it'll explicit that it's line feed stuff. Should we get rid on the asserts on the number or `read` bytes? I think it's never going to happen but technically, the contract of `read` is that it can return less than the requested number of bytes anyways so the test could be misleading?

Reply:
Sounds good. Let's keep `<= 49L` then or did you have something else in mind?

Reply:
Works for me.